### PR TITLE
added shurikens to list of available bundle items to NPC Ominous Cloud

### PR DESCRIPTION
**Pull request by tristamgreen**

### CODE REVIEW

File: scripts/zones/Port_Bastok/npcs/Ominous_Cloud.lua [hunk 1]
```diff
@@ -29,6 +29,22 @@ function onTrade(player,npc,trade)
         {2971, 5867}, -- inoshishinofuda
         {2972, 5868}, -- shikanofuda
         {2973, 5869}, -- chonofuda
+		
+		{17301, 6299}, -- shuriken
+		(17302, 6297}, -- juji shuriken
+		{17393, 6298}, -- manji shuriken
+		{17304, 6302}, -- fuma shuriken
```
Comment (ibm2431) [inline scripts/zones/Port_Bastok/npcs/Ominous_Cloud.lua]: Humbly requesting the item IDs be sorted~!

File: scripts/zones/Southern_San_dOria/npcs/Ominous_Cloud.lua [hunk 2]
```diff
@@ -29,6 +29,22 @@ function onTrade(player,npc,trade)
         {2971, 5867}, -- inoshishinofuda
         {2972, 5868}, -- shikanofuda
         {2973, 5869}, -- chonofuda
+		
+		{17301, 6299}, -- shuriken
```
Comment (TeoTwawki) [inline scripts/zones/Southern_San_dOria/npcs/Ominous_Cloud.lua]: Seems you’ve got tabs messing up the indent. Set your editor to use 4 spaces when you press the tab key or better yet, get editorconfig to automate it for you.

Comment (kaincenteno) [reply scripts/zones/Southern_San_dOria/npcs/Ominous_Cloud.lua]: @tristamgreen u should try Pycharm with lua addon, haven't had issues with indentations there 😸

Comment (tristamgreen) [reply scripts/zones/Southern_San_dOria/npcs/Ominous_Cloud.lua]: well, hopefully the newest commit fixed the tabulation errors.  i'd been without my PC for a week thanks to hurricane dorian and just got back to all this.

**Pull request merged**